config/example_config.yaml

```diff
@@ -167,7 +167,6 @@ callbacker:
       sslMode: disable
   pruneInterval: 24h # interval at which old or failed callbacks are pruned from the store
   pruneOlderThan: 336h # age threshold for pruning callbacks (older than this value will be removed)
-  quarantineCheckInterval: 1m # interval at which the store is checked for quarantined callbacks to be re-sent
-  quarantinePolicy:
-    baseDuration: 5m # initial duration a callback and its receiver is quarantined after failure
-    permQuarantineAfter: 24h # maximum time a callback can remain unsent before it's put in permanent quarantine
+  failedCallbackCheckInterval: 1m # interval at which the store is checked for failed callbacks to be re-sent
+  delayDuration: 5s # we try callbacks a few times with this delay after which if it fails consistently we store them in db
+  expiration: 24h # maximum time a callback can remain unsent before it's put as permanently failed
```

Comment:
I think the config in e2e test should be adapted as well: test/config/config.yaml